src/formats/chr.h

```diff
@@ -40,7 +40,7 @@ typedef struct {
     sd_sprite *photo;                       ///< Pilot photo
     float winnings_multiplier;              ///< Tournament winnings multiplier
     sd_chr_enemy *enemies[MAX_CHR_ENEMIES]; ///< List of enemy states in current tournament
-    unsigned int cutscene;                  ///< cutscene id for end of tournament
+    char bk_name[14];                       //< cutscene bk for end of tournament
```

Comment:
Well, guess use 3 slashes on all since the others have it too

Reply:
It's doxygen syntax. Reminds me, it has probably been a while since anyone has printed the game docs :/